src/clj/game/core/ice.clj

```diff
@@ -306,7 +306,10 @@
                                (wait-for (resolve-ability state side (make-eid state {:source-type :ability})
                                                           (break-subroutines-pay ice cost broken-subs args) card nil)
                                          (doseq [sub broken-subs]
-                                           (break-subroutine! state (get-card state ice) sub))
+                                           (break-subroutine! state (get-card state ice) sub)
+                                           (continue-ability state side (assoc (:additional-ability args)
```

Comment:
Same here, can't use `continue-ability`, we have to have the `eid` available to complete later.